src/account/PluginManager2.sol

```diff
@@ -36,19 +39,21 @@ abstract contract PluginManager2 {
             for (uint256 i = 0; i < preValidationFunctions.length; ++i) {
                 FunctionReference preValidationFunction = preValidationFunctions[i];
 
-                if (
-                    !_storage.validationData[validationFunction].preValidationHooks.add(
-                        toSetValue(preValidationFunction)
-                    )
-                ) {
-                    revert PreValidationAlreadySet(validationFunction, preValidationFunction);
-                }
+                _storage.validationData[validationFunction].preValidationHooks.push(preValidationFunction);
```

Comment:
Duplicated hooks are allowed now that this is just an array?

Reply:
This is contingent on the plugin allowing this on install though right?

Reply:
Yes, it's possible now. It depends on either the plugin defining duplicates within the manifest, or the caller of `installValidation` specifying duplicates.